### DISCUSSION TITLE
i18n Configure other paths

### DISCUSSION THREAD

**Discussion by zjy365** [facebook/docusaurus#9389] [Q&A] [answered]
Can I configure another path for i18n, not in the root directory but in the parent directory

**Comment by slorber** [comment]
Not sure what you mean but https://docusaurus.io/docs/next/api/docusaurus-config#i18n

![CleanShot 2023-10-10 at 15 36 25@2x](https://github.com/facebook/docusaurus/assets/749374/6824667f-6be6-40fe-a387-c0c2825ccdf8)

I wouldn't recommend to put `../i18n` because your docs are compiled by MDX to React components: they need to use the `./node_modules` of the current directory. If they are in parent directory they might use the wrong dependencies. If you choose to do that it's your responsibility to make it work and ensure your docs React components are using correct dependencies (you can use package manager workspace/hoisting probably but at your own risk)

**Reply by zjy365** [reply]
Thank you for your reply. Yes, I want to modify the path of the i18n folder.
<img width="350" alt="image" src="https://github.com/facebook/docusaurus/assets/72259332/3b97b2c4-884b-4c77-9818-9e01ea1da6c5">

In order to achieve the simplification of the directory structure, but it seems that the directory can not be specified in this way
<img width="1037" alt="image" src="https://github.com/facebook/docusaurus/assets/72259332/4f7df12b-ed2e-4b6d-8123-5b39c990fbaa">


But I want to get rid of docusaurus-plugin-content-docs and current

**Reply by slorber** [reply, marked as answer]
Yes it's not possible to update the folder name of the plugin/version for now, that would be a feature request
